src/AutoPilotPlugins/PX4/AirframeComponentController.cc

```diff
@@ -127,7 +127,7 @@ void AirframeComponentController::_waitParamWriteSignal(QVariant value)
         // Now that both params have made it to the vehicle we can reboot it. All these signals are flying
         // around on the main thread, so we need to allow the stack to unwind back to the event loop before
         // we reboot.
-        QTimer::singleShot(100, this, &AirframeComponentController::_rebootAfterStackUnwind);
+        QTimer::singleShot(800, this, &AirframeComponentController::_rebootAfterStackUnwind);
```

Comment:
It was definitely not, but because of the test complexity involved I might have screwed up testing. I will re-test again after my other fixes with the original (or lower) number.